classes/merge_request.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Merge user accounts
+ * @package   tool_mergeusers
+ * @author    Nicola Vallinoto <n.vallinoto@liguriadigitale.it>
+ * @copyright 2023 Liguria Digitale (https://www.liguriadigitale.it)
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+namespace tool_mergeusers;
+/**
+ * Merge request definition.
+ *
+ * @package   tool_mergeusers
+ * @author    Nicola Vallinoto <n.vallinoto@liguriadigitale.it>
+ * @copyright 2023 Liguria Digitale (https://www.liguriadigitale.it)
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * */
+class merge_request {
+      /**
+     * Missing merging request id This id really does not exist.
+     * @var integer
+     */
+    const MISSING_MERGING_REQUEST_ID = 1;
+    /**
+     * Merging request queued but not yet processed.
+     * @var integer
+     */
+    const QUEUED_NOT_PROCESSED = 2;
+    /**
+     * Merging request queued, to be processed (the adhoc task is created for it).
+     * @var integer
+     */
+    const QUEUED_TO_BE_PROCESSED = 3;
+    /**
+     * Merging request is in progress but not concluded (its adhoc_task has started).
+     * @var integer
+     */
+    const INPROGRESS_NOT_CONCLUDED = 4;
+    /**
+     * Tried with error; pending of retrial (the merging process has found some error
+     * and the adhoc_task is queued to be retried in the future).
+     * @var integer
+     */
+    const TRIED_WITH_ERROR = 5;
+    /**
+     * Completed with success.
+     * @var integer
+     */
+    const COMPLETED_WITH_SUCCESS = 6;
+    /**
+     * Completed with errors (even with all the current retries the merging process
+     * could not end with success).
+     * @var integer
+     */
+    const COMPLETED_WITH_ERRORS = 7;
+    /**
+     * Reference table for merge requests.
+     */
+    const TABLE_MERGE_REQUEST = 'tool_mergeusers_queue';
+     /**
+     * Reference table for merge requests. Before web service integration.
+     */
+    const TABLE_MERGE_REQUEST_OLD = 'tool_mergeusers';
+    /**
+     * Reference table for users.
+     */
+    const TABLE_USERS = 'user';
+    
```

Comment:
how to do it ?

Reply:
you mean this comment ?
// This file is part of Moodle - http://moodle.org/
//
// Moodle is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// Moodle is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.

Reply:
I meant just:

```php
/**
 * @var {type} {description}
 */
private $data;
```

Reply:
ok. already done-